stats/lib/handler.js

```diff
@@ -62,29 +73,30 @@ export const sanitizePathname = pathname => `/${pathname.split('/').filter(Boole
  */
 const handler = async (req, res, pgPools) => {
   // Caveat! `new URL('//foo', 'http://127.0.0.1')` would produce "http://foo/" - not what we want!
-  let { pathname, searchParams } = new URL(`http://127.0.0.1${req.url}`)
-  pathname = sanitizePathname(pathname)
+  const { pathname, searchParams } = new URL(`http://127.0.0.1${req.url}`)
+  const segs = pathname.split('/').filter(Boolean)
+  const url = `/${segs.join('/')}`
 
   enableCors(req, res)
   const respond = createRespondWithFetchFn(pathname, searchParams, res, pgPools)
```

Comment:
I think we should pass the normalised pathname (the value stored in `url`) to the `pathname` argument. That was the behaviour before your change, if I am reading the code correctly.

The same comment applies to `stats/lib/platform-routes.js`.

Thoughts?

Reply:
I think you're right, tests didn't catch it though. Will follow up

Reply:
https://github.com/filecoin-station/spark-stats/pull/180